pages/mesosphere/dcos/services/edge-lb/1.5/concepts/how-edge-lb-works/index.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: layout.pug
+navigationTitle: How Edge-LB works
+title: How Edge-LB works
+menuWeight: 10
+excerpt: Highlights the basic workflow and key components for Edge-LB operations
+enterprise: true
+---
+
+Edge-LB is designed to handle load balancing and proxying for a distributed architecture.
+It is usually deployed on the **public agent nodes** of a cluster so that the services running on the cluster remain protected inside of a security perimeter. The public agent nodes are typically in demilitarized zones (DMZs) or in perimeter external networks that are outside of the DC/OS cluster and your organizational WAN, LAN, or VPN networks.
+
+In general, you should deploy Edge-LB on **multiple public nodes** to support high availability for traffic entering the cluster.
+
+# Key Edge-LB components
+Edge-LB relies on the following core components to provide proxy and load-balancing service for inbound traffic to the DC/OS cluster:
+- Edge-LB API server
+- Edge-LB pool
+- Individual load balancer
+
+## Edge-LB API server
+
+The Edge API server enables you to create, modify, and delete Edge-LB pools and their associated pool configuration settings.
+Essentially, the Edge-LB API server provides load balancing orchestration service for traffic entering the cluster.
+The Edge-LB API server:
+- Performs create, read, update, and delete (CRUD) actions on Edge-LB pool(s).
+- Responds to Edge-LB pool-related API endpoint queries.
+- Stores the pool configuration in a configuration management system and automatically updates and reloads pool configuration changes .
```

Comment:
WRT steps 2 and 4, is your point that these are not the same directories? (I am still very new to D2iQ and do not know what the default working directory is or should be.)

Reply:
I included the links in the attached document - thank you very much! I cannot test themas my D2iQ account does not permit me to see any downloads. If you have time, it would be super-helpful to know if the links resolve properly for you.